### PR TITLE
samples: Add Prediction CPR preprocess sample.

### PR DESCRIPTION
**Pull request by abcdefgs0324**
If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [X] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb [hunk 1]
```diff
@@ -0,0 +1,1744 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb:1]: <p>use print</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='91'/>

Comment (andrewferlitsch) [inline notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb:1]: <p>Add what to expect in logs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='92'/>

Comment (andrewferlitsch) [inline notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb:1]: <p>use print</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>

File: notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb [hunk 2]
```diff
@@ -0,0 +1,1773 @@
...
```
Comment (andrewferlitsch) [inline notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb:1]: <p>Replace all occurrences of "we" with "you"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (andrewferlitsch) [inline notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb:1]: <p>The preprocessor will then be used -&gt; The processor is then used</p><p> </p><p>**Avoid future tense**</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Comment (andrewferlitsch) [inline notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb:1]: <p>I see another "we" here. Replace throughout notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

File: notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (abcdefgs0324) [reply notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb:1]: The instruction in https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/ml_ops/stage1/get_started_with_data_labeling.ipynb shows there's no any work in Workbench and users only need to populate their email.

However, if we need to generate a credential file, we need to follow what we have in our notebooks. I would prefer leaving it as is.

[the remark this answers is not in the document]
Comment (andrewferlitsch) [reply notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb:1]: <p>ok</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**